org.eclipse.xtext.testing/META-INF/MANIFEST.MF

```diff
@@ -27,7 +27,9 @@ Require-Bundle: org.eclipse.xtext;visibility:=reexport,
  org.eclipse.xtend.lib
 Import-Package: org.apache.log4j;version="1.2.15",
  org.apache.log4j.spi;version="1.2.15",
- org.junit.jupiter.api.extension;version="[5.0.0,6.0.0)";resolution:=optional
+ org.junit.jupiter.api.extension;version="[5.0.0,6.0.0)";resolution:=optional,
+ org.junit.platform.runner;version="[1.0.0,2.0.0)";resolution:=optional,
```

Comment:
Not needed. I'll remove that since it's only required for tests and not for consumers of this bundle.